### ISSUE TITLE
Ghosting and Suicide

### ISSUE THREAD

**Issue by nuklearcellphoneg** [d3athrow/vgstation13#13463]
tl;dr version

Suiciding should be limited to certain items. It would be easier to suicide with a gun, low chance with a air bubble needle because that's horrifying, and biting your tongue off, if even still a thing, should have a very low chance and looooong cool down after attempting.

Ghosting should be reserved for people who are dead.

Instead of deathgasp, give everyone one statement per crit session and limit the characters(maybe on the 2nd part). .

1. Normal free people could still very easily suicide and ghost if they want to. Plastic bags, needles, etc are plentiful or they can jump out an airlock.

2. Nonantag greytide shitters wouldn't have access to suicide because of these limited items, when put in perma or jail- because of this, they can't suicide, ghost, then rejoin as a borer, borg, etc 

3.  If above causes sec to perma over execute more often, antags will be perma'd more often making jailbreaks for feasible as long as you don't commit too extreme of a crime. This helps encourage riots, breaking out of perma, etc, Perma is not a perfect prison.

4. It would cause people who are converted or kidnapped to have a more difficult time ghosting or suiciding. This would encourage salty converts to at least think about suiciding,  especially if they have to attempt it multiple times and would prevent a kidnap victim from simply ghosting rerolling a secondary spawn(pai borg etc).
------------------------------
This causes a wide range of issues but I'll stick with the obvious and well known. Security arrests a greytiding shitter... or heck, an antag. They ghost. Or they suicide. Then they ghost. A lot of times people ghost when things don't go their way, and that can lead to admins and players at odds as well as divided on what to do- like if someone converts someone to rev/thrall/cult/greytide et al, and then they ghost, we have to punish them.

This does no one any favors and everyone loses out. 

If you punish someone for  ghosting after conversion, antags lose their convert, admin and the player is bitter, etc. and the player may be vaguely unaware of why this is bad, or blinded by salt because they 'lost'.

Rather, if you removed the ability to magically ghost or suicide while alive, this could be curbed. Consider the concept of using a game's design to teach a concept. In this case, since they can't simply suicide and ghost via quickly typing out a frustrating word, 'Well fuck I am might as well help them.' 

Now as it is, you can get special suicide flavor text via having certain items in hand, right? Okay, so people would obviously be upset at losing the ability to suicide- relegate it to the aforementioned items being necessary to do so. This way, suicide is an option, but requires forethought and a smidgen of prep.

Oh, but you can suicide with needles, bags, guns, it will be easy for people to get these, right?

Set a probability for each item to be successful. A gun would be fairly easy to use to kill yourself- it's just a trigger pull, after all). Suffocating yourself with a bag? Low chance to work. Air bubble in a needle? That sounds like a spooky way to die.

So, back to the bit about jail. It's possible to escape from Perma. So, antags being put in Perma is a good thing- like if you arrest five cultists and put them in perma, they can totes do a prison riot. People can break in, they can break out yadda yadda, antag in general.

Consider the greytide: Not restricted to assistants necessarily, people who just muck about and break laws and eventually either get executed or perma'd, clown, mime, assistant, and so on. If they grey tide a bunch and get caught, whoopedy doo perma: suicide-ghost. However, if in perma they didn't have anything with which to kill themselves, they would either have to sit around, break out, or go do something else- something other than just roll respawn to be a borer, diona, xeno, wizard, borg, pai, y'know, whatever is available really.

Of course, if your average joe wanted to commit suicide and ghost, they could- if they got frustrated nibbling their tongue they could always leap out an airlock. But it would be a slightly more deliberate thing, and in some cases ghosting/suicide could be preventable- similarly, if someone kidnaps someone, they might ghost, and this would prevent that. Because, assuming someone doesn't outright kill someone else and have made an effort to stun and restrain them, they are actively restraining their ability to commit suicide.

**Comment by Shadowmech88** [comment]
What's to stop people from just punching themselves until they enter crit and then waiting until the suffocation damage kills them?

**Comment by nuklearcellphoneg** [comment]
I think most of the time, the immediacy of suicide/ghost gratification is what makes it so knee jerk. In the case of converts and kidnapping, they are likely restrained. It would take awhile to punch yourself to death- and I think the amount of time that would take would allow them to think 'ah you know i'm kind of a being a jackass'.

It takes effort to punch yourself to death, ghosting not so much.

The case of perma, I kind of see that but hey- if they are a nonantag, bed cuff them, heal them, use the  strait jacket- because, in this sense, the criminal really is paying for the crime by making things as uninteresting as possible, like real jail- can't just beat yourself to death and observe kiddo, let alone respawn as something. I think this would help to police greytide and sec and conversions better via IC methods allowing for better immersion all around, because I honestly think prisoners and converts et al magically ghosting is frustrating and immersion breaking for all parties involved.

**Comment by nuklearcellphoneg** [comment]
I would say this would be a huge change, but at the same time, would largely not take anything away from unrestrained free crew members.

**Comment by Kurfursten** [comment]
You can shard yourself to death pretty fast in perma

**Comment by nuklearcellphoneg** [comment]
And stools.

**Comment by nuklearcellphoneg** [comment]
Again though, I'm sure regular security players are familiar with people ghosting often and would gladly keep an extra watchful eye  on their prisoners health

**Comment by Shadowmech88** [comment]
>In the case of converts and kidnapping, they are likely restrained.
>if they are a nonantag, bed cuff them, heal them, use the strait jacket-

Are you aware that being restrained already prevents you from using the suicide verb?
How many of the problems you've highlighted could be fixed by simply preventing ghosting while restrained as well?

**Comment by nuklearcellphoneg** [comment]
![image](https://cloud.githubusercontent.com/assets/6540760/21956278/5eae71f0-da4b-11e6-9369-debe0b51328d.png)

**Comment by Shadowmech88** [comment]
I'm not sure what you're trying to say with that.
I never claimed that being restrained prevents you from ghosting.

**Comment by Kammerjunk** [comment]
Nuke that's exactly what he said 😕

**Comment by ghost** [comment]
Make it a do_after so you can stop some dumbass from shoving a screwdriver through his temple, rather than it being instantaneous

It works with guns, why not anything else?

Hell, like nuke said, some shouldn't be a guarantee or an instantanous death. Biting down on your tongue doesn't kill you right away, it just means you're likely to bleed out or drown in your own blood.

**Comment by nuklearcellphoneg** [comment]
Yeah I realized. You're right, ghosting is the more prevalent and powerful bit, by far. 

Consider when someone literally suicides for a small 5 minute sentence, or when deconverted in Security, and other similar situations- I think those are problematic and both suicide and ghosting are symptomatic of the other's issues, and suicide would just be a stepping stone for  ghosting.

**Comment by nuklearcellphoneg** [comment]
Removing ghosting while alive would alleviate immediately and fix some, from a meta perspective, situations that are black and white(cuffed... shouldn't be able to do that).

Suicide itself is trickier(because even if you removed it for some reason there are alternate ways of doing so the hard way) and I think removing it would be bad. I think adding an extra layer of effort to suicide, however, would help give way to rational thought in some situations, and again, allow sec to actually enforce their punishments.

**Comment by Exxion** [comment]
How about make it so that suicide requires a sufficient item, leave the ability to ghost how it is, but make it so that if you ghost while alive, you cannot re-enter the round by ANY means?
Because forcing someone to lie permanently restrained in one place for the rest of the round is shitty

**Comment by Shadowmech88** [comment]
>Because forcing someone to lie permanently restrained in one place for the rest of the round is shitty

And yet, the victim deserves the punishment of not being able to re-enter the round if they decide they don't want to sit there forever?

**Comment by nuklearcellphoneg** [comment]
Okay, so like in the case of antags, I am fairly confident any antag worth his or her salt has the capability of breaking out of perma albeit with some forethought, but even without planning. They can kill whereas an nonantag would have to manage to get out without killing, since they got themselves in there in the first place. In the case of team antags, you got buddies willing to crack open perma.

In the case of nonantags who have committed enough crimes to be placed in perma?  If they try to kill themselves, yeah, tie them down.  Don't get put in perma in the first place. Because if they continue greytide and get in perma, they'll wind up banned... which is a lot like laying restrained in perma.

**Comment by Shadowmech88** [comment]
What if someone takes your radio and walls you in maint in a straitjacket?
Or fills you up with hundreds of units of sleep toxin?
You'll either have to sit there forever or ghost, and if you ghost, you wouldn't ever be able to get back in the round.

**Comment by Exxion** [comment]
@Shadowmech88 It's better than not even being able to observe.

**Comment by Aurx** [comment]
History lesson time!  
Way back when, there was no such thing as a "ghost" verb and you could suicide while restrained.  

Butthurt at shitters who suicided when arrested, a coder changed things to the way they are now.  
The shitters just switched verbs and everything carried on as it was.  

Making this change would mean the shitters would log out, provoke sec into killing them (and maybe ahelping it after if they were turboshitters), or pester the 'mins to kill/free them.  

The correct change is to make suicides and those who ghosted while alive unable to join the recruit lists for alien/borer/pai/etc etc etc.

**Comment by Aurx** [comment]
Correction.  The correct change _IF this is somehow both an issue that needs to be dealt with AND somehow a code issue instead of an administrative issue_ is to make suicides and those who ghosted while alive unable to join the recruit lists for alien/borer/pai/etc etc etc.
